packages/jest-diff/src/diffStrings.js

```diff
@@ -68,23 +76,24 @@ const diffLines = (a: string, b: string): Diff => {
   };
 };
 
+const showPatchMarks = (hunk: Hunk, a: string): boolean => {
+  const oldLines: Array<string> = a.match(/\n/g) || [];
+  return oldLines.length > hunk.oldLines;
```

Comment:
why are you comparing to `hunk.oldLines` here and not `newLines`? Can you explain this a bit better?